Google.PowerShell.IntegrationTests/Logging/Logging.GcLogSink.Tests.ps1

```diff
@@ -72,3 +69,347 @@ Describe "Get-GcpsLogSink" {
         { Get-GcLogSink -Sink "non-existent-sink-name" -ErrorAction Stop } | Should Throw "does not exist"
     }
 }
+
+Describe "New-GcpsLogSink" {
+    It "should work with -GcsBucketDestination" {
+        $r = Get-Random
+        $bucket = "gcloud-powershell-testing-logsink-bucket-$r"
+        $sinkName = "gcps-new-gclogsink-$r"
+        try {
+            New-GcLogSink $sinkName -GcsBucketDestination $bucket
+            Start-Sleep -Seconds 1
```

Comment:
Start-Sleep actually does not use Thread.Sleep underneath. It uses ManualResetEvent.WaitOne